Dockerfile

```diff
@@ -11,11 +11,15 @@ ENV PATH="/opt/depot_tools:${PATH}"
 
 RUN mkdir /root/repo
 WORKDIR /root/repo
-ARG revision=pandafium
-RUN gclient config --unmanaged --name pdfium https://github.com/gradescope/pdfium.git@origin/$revision
-RUN gclient sync
+
+RUN gclient config --unmanaged --name pdfium https://github.com/gradescope/pdfium.git
+ADD . / pdfium/
```

Comment:
This copies in the current working directory. From https://stackoverflow.com/a/42249980